yarn.lock

```diff
@@ -1696,19 +1696,27 @@
 
 "@types/enzyme-adapter-react-16@^1.0.6":
   version "1.0.6"
-  resolved "https://registry.yarnpkg.com/@types/enzyme-adapter-react-16/-/enzyme-adapter-react-16-1.0.6.tgz#8aca7ae2fd6c7137d869b6616e696d21bb8b0cec"
-  integrity sha512-VonDkZ15jzqDWL8mPFIQnnLtjwebuL9YnDkqeCDYnB4IVgwUm0mwKkqhrxLL6mb05xm7qqa3IE95m8CZE9imCg==
+  resolved "https://artifactory.rnd-hub.com:443/artifactory/api/npm/Npm-IMFamily-virt/@types/enzyme-adapter-react-16/-/enzyme-adapter-react-16-1.0.6.tgz#8aca7ae2fd6c7137d869b6616e696d21bb8b0cec"
```

Comment:
please remove `rnd-hub` artifactory and use `yarnpkg`

Reply:
fixed in 3147b03